cmd/release-controller/sync_verify_prow.go

```diff
@@ -152,18 +152,45 @@ func addReleaseEnvToProwJobSpec(spec *prowjobv1.ProwJobSpec, release *Release, m
 			}
 		}
 		if !hasReleaseImage {
-			c.Env = append(c.Env, corev1.EnvVar{Name: "RELEASE_IMAGE_LATEST", Value: release.Target.Status.PublicDockerImageRepository + ":" + releaseTag.Name})
+			switch architecture {
+			case "arm64":
+				c.Env = append(c.Env, corev1.EnvVar{Name: "RELEASE_IMAGE_ARM_LATEST", Value: release.Target.Status.PublicDockerImageRepository + ":" + releaseTag.Name})
+			case "s390x":
+				c.Env = append(c.Env, corev1.EnvVar{Name: "RELEASE_IMAGE_Z_LATEST", Value: release.Target.Status.PublicDockerImageRepository + ":" + releaseTag.Name})
```

Comment:
These env var names should 1:1 match the architecture string:

```
RELEASE_IMAGE_S390_LATEST
RELEASE_IMAGE_ARM64_LATEST
```

unless there is a reason for them to be different.

Reply:
@deepsm007 Was there a specific reason you chose the names that you did?

Reply:
There was no specific reason